### PR TITLE
New weights for InShop and SOP

### PR DESCRIPTION
**Pull request by DaloroAT**

### CODE REVIEW

File: oml/models/vit/vit.py [hunk 1]
```diff
@@ -38,13 +38,16 @@ class ViTExtractor(IExtractor):
         "vitb8_dino": (f"{_FB_URL}/dino/dino_vitbase8_pretrain/dino_vitbase8_pretrain.pth", "556550", None),
         # our pretrained checkpoints
         "vits16_inshop": (
-            [f"{_STORAGE_CKPTS}/inshop/vits16_inshop.ckpt", "1wjjwBC6VomVZQF-JeXepEMk9CtV0Nste"],
-            "e1017d",
+            [
+                f"{_STORAGE_CKPTS}/inshop/vits16_inshop_a76b85.ckpt",
+                "1wjjwBC6VomVZQF-JeXepEMk9CtV0Nste",
+            ],  # UPDATE GDRIVE!
```
Comment (DaloroAT) [inline oml/models/vit/vit.py:44]: upload to gdrive and update

File: oml/models/vit/vit.py [hunk 2]
```diff
@@ -38,13 +38,16 @@ class ViTExtractor(IExtractor):
...
+            "a76b85",
             "vits16_inshop.ckpt",
         ),
         "vits16_sop": (
-            [f"{_STORAGE_CKPTS}/sop/vits16_sop.ckpt", "1IXDQoHUCDIcpyKMA_QrcyXdz3dXaYXCt"],
-            "85cfa5",
+            [f"{_STORAGE_CKPTS}/sop/vits16_sop_21e743.ckpt", "1IXDQoHUCDIcpyKMA_QrcyXdz3dXaYXCt"],  # UPDATE GDRIVE!
```
Comment (DaloroAT) [inline oml/models/vit/vit.py:49]: upload to gdrive and update

Comment (DaloroAT) [reply oml/models/vit/vit.py]: done

File: oml/models/vit/vit.py (hunk 1 above)
Comment (DaloroAT) [reply oml/models/vit/vit.py]: done

**Review by AlekseySh — APPROVED** [review]
Super! New SotA! :)

**Pull request merged**